freqtrade/plugins/pairlist/RemotePairList.py

```diff
@@ -57,6 +58,23 @@ def __init__(self, exchange, pairlistmanager,
                 '`mode` not configured correctly. Supported Modes '
                 'are "whitelist","blacklist"')
 
+        if self._processing_mode not in ['filter', 'append']:
+            raise OperationalException(
+                '`processing_mode` not configured correctly. Supported Modes '
+                'are "filter","append"')
+
+        pairlists = self._config['pairlists']
+
+        if len(pairlists) == 1 and self._mode == 'blacklist':
+            raise OperationalException(
+                'At `blacklist` mode RemotePairList requires an additional '
+                'Pairlist and cannot be used on its own.')
+
+        if pairlists[0]['method'] == "RemotePairList" and self._mode == 'blacklist':
+            raise OperationalException(
+                'At `blacklist` mode RemotePairList can not be on the first '
+                'position of your pairlist.')
```

Comment:
You should be able to use `pairlist_pos` to check for this in one go
if the position is 0, it's the first pairlist - and can't be used in blacklist mode.

the "on it's own" check is really irrelevant then if you think about it.